test/ui-schema-gen.test.ts

```diff
@@ -0,0 +1,344 @@
+import test from 'ava';
+import generateDefaultUISchema, { startCase } from '../src/generators/ui-schema-gen';
+import { JsonSchema } from '../src/models/jsonSchema';
+import { Layout, ControlElement } from '../src/models/uischema';
+
+test('startCase', t => {
```

Comment:
Ironically, the test case failures I had after porting the generator were all related to labels ;)
So, I'd argue to just leave it as is.